### PR TITLE
V9_4-HTCONDOR-794-branch

### PR DESCRIPTION
**Pull request by Todd-L-Miller**
Sorry about forgetting to actually create this PR.  This PR implements the submit file command `allowed_job_duration`, as we discussed on Monda.

### CODE REVIEW

File: docs/classad-attributes/job-classad-attributes.rst [hunk 1]
```diff
@@ -42,6 +42,14 @@ all attributes.
     is only present if an accounting group was requested by the
     submission. :index:`AllRemoteHosts<single: AllRemoteHosts; ClassAd job attribute>`
```
Comment (JaimeFrey) [inline docs/classad-attributes/job-classad-attributes.rst:43]:
```suggestion
    submission.
```

**Comment by tannenba** [comment]
Comments:

1. I would like to see this implemented just via ShadowBirthDate, since it is a) less code/complexity and does not introduce yet another (nearly duplicate) attribute, b) allows support for run limits for many other universes and Jaime stated. 
2. Suggest the check be moved into the UserPolicy class in the util library, so all these other universes work "for free".
3. One advantage of making this first class is we can do better with the frequency of checks than just the usual job policy checks.  IIRC, the shadow only calls the user policy eval function when it receives an update from the condor_starter (which is every 5 min by default, but could be configured to be a much longer interval).  To improve upon this, I suggest the Shadow also set a timer to eval the policy for 1 second beyond when we think the max claim time has been exhausted.
4. The documentation is very developer/expert, using expert lingo lingo like maximum claim activation time.  This is fine to include, but first it should give a layperson description ("The maximum time (in seconds) a job is allowed to occupy a slot, including both run time and also file transfer time....").

File: docs/man-pages/condor_submit.rst [hunk 2]
```diff
@@ -1404,6 +1404,17 @@ FILE TRANSFER COMMANDS
...
 
+ allowed_job_duration = <integer>
+    The longest time for which a job may continuously be in the running state.
+    Jobs which exceed this duration will go on hold.  Exiting the running
+    state resets the job duration used by this command.
+
+    This command is intended to help minimize the time wasted by jobs
+    which may erroneously run forever.
+
+    This command currently only functions for the vanilla, docker,
```
[the remark this answers is not in the document]
Comment (Todd-L-Miller) [reply docs/man-pages/condor_submit.rst:1415]: I'd rather document only the universes we actually test, but I guess we can decide to change the documentation or the code at the time if anybody ever actually complains.

**Review by JaimeFrey — APPROVED** [review]
LGTM

**Pull request merged**